rollup/tracing/tracing.go

```diff
@@ -23,6 +26,21 @@ import (
 	"github.com/scroll-tech/go-ethereum/trie/zkproof"
 )
 
+type TracerWrapper struct{}
```

Comment:
ditto, comment

Reply:
resolved in https://github.com/scroll-tech/go-ethereum/pull/615/commits/acf23b72b5312edadf3ff4a9a1f6a3248ab47d75